shared/src/components/exercise/mode.cjsx

```diff
@@ -31,7 +31,7 @@ ExMode = React.createClass
 
   componentDidUpdate: (prevProps) ->
     {mode, focus} = prevProps
-    @setFreeResponseFocusState() if mode is 'free-response' and focus isnt @props.focus
+    @setFreeResponseFocusState() if mode is 'free-response'
```

Comment:
ooh i see, it's not changing when the question updates, you could test props for question changes